### ISSUE TITLE
Pluralsight Unable to log in

### ISSUE THREAD

**Issue by jeehay** [ytdl-org/youtube-dl#18976]
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['--username', 'PRIVATE', '--password', 'PRIVATE', '-o', '%(playlist_index)s - %(title)s.%(ext)s', '--playlist-start', '1', '--verbose', '--min-sleep-interval', 
'90', '--max-sleep-interval', '130', 'https://app.pluralsight.com/library/courses/xxxx/table-of-contents']
[debug] Encodings: locale cp1252, fs mbcs, out cp1252, pref cp1252
[debug] youtube-dl version 2019.01.24
[debug] Python version 3.4.4 (CPython) - Windows-10-10.0.17763
[debug] exe versions: none
[debug] Proxy map: {}
[pluralsight:course] xxxx: Downloading JSON metadata
[download] Downloading playlist: xxxxxx
[pluralsight:course] playlist xxxx 59 video ids (downloading 59 of them)
[download] Downloading video 1 of 59
[pluralsight] Downloading login page
[pluralsight] Logging in
ERROR: Unable to log in; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the 
--verbose flag and include its complete output.
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpa8aoq3sz\build\youtube_dl\YoutubeDL.py", line 793, in extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpa8aoq3sz\build\youtube_dl\extractor\common.py", line 507, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpa8aoq3sz\build\youtube_dl\extractor\common.py", line 411, in initialize
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpa8aoq3sz\build\youtube_dl\extractor\pluralsight.py", line 165, in _real_initialize
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpa8aoq3sz\build\youtube_dl\extractor\pluralsight.py", line 215, in _login
youtube_dl.utils.ExtractorError: Unable to log in; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to 
call youtube-dl with the --verbose flag and include its complete output.

**Comment by jeehay** [comment]
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['--username', 'PRIVATE', '--password', 'PRIVATE', '-o', '%(playlist_index)s - %(title)s.%(ext)s', '--playlist-start', '1', '--verbose', '--min-sleep-interval', 
'90', '--max-sleep-interval', '130', 'https://app.pluralsight.com/library/courses/windows-server-2016-active-directory-certificate-services/table-of-contents']
[debug] Encodings: locale cp1252, fs mbcs, out cp1252, pref cp1252
[debug] youtube-dl version 2019.01.24
[debug] Python version 3.4.4 (CPython) - Windows-10-10.0.17763
[debug] exe versions: none
[debug] Proxy map: {}
[pluralsight:course] windows-server-2016-active-directory-certificate-services: Downloading JSON metadata
[download] Downloading playlist: Implementing Active Directory Certificate Services in Windows Server 2016
[pluralsight:course] playlist Implementing Active Directory Certificate Services in Windows Server 2016: Collected 79 video ids (downloading 79 of them)
[download] Downloading video 1 of 79
[pluralsight] Downloading login page
[pluralsight] Logging in
ERROR: Unable to log in; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the 
--verbose flag and include its complete output.
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpa8aoq3sz\build\youtube_dl\YoutubeDL.py", line 793, in extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpa8aoq3sz\build\youtube_dl\extractor\common.py", line 507, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpa8aoq3sz\build\youtube_dl\extractor\common.py", line 411, in initialize
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpa8aoq3sz\build\youtube_dl\extractor\pluralsight.py", line 165, in _real_initialize
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpa8aoq3sz\build\youtube_dl\extractor\pluralsight.py", line 215, in _login
youtube_dl.utils.ExtractorError: Unable to log in; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to 
call youtube-dl with the --verbose flag and include its complete output.

**Comment by Tarby2019** [comment]
FWIW I used this very function in late Jan, early Feb 2019 to download a few serieses of videos.  I logged in and everything, I think using the same syntax as the OP.  I doubt it's relevant, but I didn't pass any arguments like playlist-start of min-sleep-interval.  Also I don't know that the URLs I passed ended with the table-of-contents.

I don't guess the OP has any odd characters in their username or password?  Maybe something that's usually encoded as an HTML entity?
